pkg/apis/performanceprofile/v2/performanceprofile_validation.go

```diff
@@ -344,3 +345,25 @@ func (r *PerformanceProfile) validateWorkloadHints() field.ErrorList {
 	}
 	return allErrs
 }
+
+func (r *PerformanceProfile) validateCpuFrequency() field.ErrorList {
```

Comment:
I think this should also verify that the user doesn't set one of the new fields but not both. That should be treated as an error as both are required for the new feature to work.

Reply:
Added that verification

Reply:
thanks - looks good